lib/java/test-utils/src/main/java/org/enso/test/utils/ContextUtils.java

```diff
@@ -408,12 +410,48 @@ private static Context.Builder defaultContextBuilder(String... permittedLanguage
               Paths.get("../../distribution/component").toFile().getAbsolutePath());
     }
 
+    private static Context.Builder defaultContextBuilder(
+        Level logLevel, Handler logHandler, String... permittedLanguages) {
+      return Context.newBuilder(permittedLanguages)
+          .allowExperimentalOptions(true)
+          .allowIO(IOAccess.ALL)
+          .allowAllAccess(true)
+          .environment("NO_COLOR", "true")
+          .logHandler(logHandler)
+          .option(RuntimeOptions.LOG_LEVEL, logLevel.getName())
+          .option(RuntimeOptions.DISABLE_IR_CACHES, "true")
+          .option(RuntimeOptions.STRICT_ERRORS, "true")
+          .option(
+              RuntimeOptions.LANGUAGE_HOME_OVERRIDE,
+              Paths.get("../../distribution/component").toFile().getAbsolutePath());
+    }
+
     /** Modifies the {@link Context.Builder}. Can be called multiple times. */
     public Builder withModifiedContext(Function<Context.Builder, Context.Builder> modifier) {
       polyglotCtxBldr = modifier.apply(polyglotCtxBldr);
       return this;
     }
 
+    /**
+     * Shortcut for {@code withModifiedContext(b -> b.option(RuntimeOptions.LOG_LEVEL,
+     * logLevel.getName()).logHandler(logHandler))}.
```

Comment:
- Could this behavior be the default?
- And calling `withLogHandler(null, null)` or co. would be an opt-out?

Reply:
I second that. I would also like the default log handler the one that consumes everything and only prints it on failure.